### PR TITLE
Home Page for the Project

### PR DESCRIPTION
**Pull request by rna**
### Home Page for the project with the following accomplished tasks: 
* Header for the page with search & sign-in in the header
* Main video section for the page with channel and video information
* Recommended videos in the sidebar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                        <li><i class="fa fa-thumbs-up" aria-hidden="true"></i> 359</li>
+                        <li><i class="fa fa-thumbs-down" aria-hidden="true"></i> 14</li>
+                    </ul>
+                </div>
+               </div>
+               <div id="description">
+                    <h4>Published on Feb 27, 2014</h4>
+                    <p width="50vw">How do you contribute to open source? In this video, I'll describe the easiest way to make a simple contribution to an open source project -- without using the command line, a text editor, "forking a repo", or even touching Git outside of Github. I'll also describe what kinds of contributions open source software projects are typically looking for. As an example, I use submitting a student solution to a web development exercise from The Odin Project, a free online curriculum for learning web development with Ruby on Rails. You can find more information about that project at http://theodinproject.com.</p>
+                    <hr>
+                    <div style="text-align:center"><a href="">SHOW MORE</a></div>
```
[the remark this answers is not in the document]
Comment (rna) [reply index.html]: Issue fixed

File: index.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                    </div>
+                    <div>
+                        <h3>Descriptive Title of the youtube video</h3>
+                        <p>Channel Name</p>
+                        <p>981,643 views</p>
+                    </div>
+            </div>
+            <div>
+                <hr>
+                <div style="text-align:center"><a href="">SHOW MORE</a></div>
```
[the remark this answers is not in the document]
Comment (rna) [reply index.html]: Issue fixed

**Comment by rna** [comment]
Fixed and modified as per suggestions.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css">
+        <link rel="stylesheet" href="./styles.css">
+    </head>
+    <body>
+        <!--Begining of Navigation Bar-->
+        
+        <header id="navigation-bar">
+            
+                <a href=""><i class="fa fa-bars fa-lg" aria-hidden="true"></i></a>
+                </button>
```
Comment (Redvanisation) [inline index.html]: Remove is this closing `</button>` tag

Comment (rna) [reply index.html]: Removed.

**Pull request merged**